lessons/introR/lesson.md

```diff
@@ -109,10 +110,10 @@ datasets that you can practice on.  The one I'm going to use is the
 `swiss` dataset.
 
 
-~~~r
```

Comment:
Yea not sure why that happened. I think I accidentally changed the code to create this. Imma fix this next week (busy right now). But you're right, this isn't necessary

On 9 June, 2016 8:32:36 PM EDT, Elliott Sales de Andrade <notifications@github.com> wrote:
>> @@ -109,10 +110,10 @@ datasets that you can practice on.  The one I'm
>going to use is the
>>  `swiss` dataset.
>>  
>>  
>> -~~~r
>
>OK, maybe this is where I should ask: Is this necessary? It appears
>that the existing pages is formatted correctly, no?
>
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/UofTCoders/studyGroup/pull/141/files/b8607f847c3a48470f33a71b07e56644e865b408#r66545525

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.